.gitignore

```diff
@@ -1,4 +1,14 @@
+# if you want to ignore files created by your editor/tools,
+# please consider a global .gitignore https://help.github.com/articles/ignoring-files
+*.exe
+*.exe~
+*.orig
+.*.swp
 .DS_Store
+Thumbs.db
+# a .bashrc may be added to customize the build environment
+.bashrc
```

Comment:
The custom .bashrc only works in the context of the _moby/moby_ build scripts: https://github.com/moby/moby/blob/master/Dockerfile#L179. There's no corresponding feature in this repo.

Reply:
Ah, yes, makes sense, I can remove that, thanks 👍